script/foreman-debug

```diff
@@ -349,7 +349,7 @@ add_files /var/log/foreman/apipie_cache*.log*
 add_files /var/log/foreman/cron*.log*
 add_files /var/log/foreman/db_migrate*.log*
 add_files /var/log/foreman/db_seed*.log*
-add_files /var/log/foreman/production-*.log*
+add_files /var/log/foreman/production.log*
 add_files /var/log/foreman/production.log
```

Comment:
Damn, this will capture `production.log` as well, we actually don't want to do that. Just use this please:

    add_files /var/log/foreman/production.log-*